### PR TITLE
hi

### PR DESCRIPTION
**Pull request by 5rq**
hi

### CODE REVIEW

**Comment by omxpro** [comment]
Why there is no repo url?

**Comment by 5rq** [comment]
> Why there is no repo url?

good question
I forgot it 🤷‍♂️

File: domains/syz.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+   "owner":{
+      "username":"Syz",
```
Comment (MaskDuck) [inline domains/syz.json:3]: your gh username?

File: domains/syz.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+      "discord":"Syz#0128",
+      "email":"Syz@myself.com"
+      "repo":"https://github.com/5rq/5rq.github.io"
```
Comment (MaskDuck) [inline domains/syz.json:6]: repo should be a dedicated field instead

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by omxpro** [comment]
F

**Pull request closed without merging**